mlflow/store/file_store.py

```diff
@@ -367,7 +367,9 @@ def _get_param_from_file(parent_path, param_name):
     def _get_tag_from_file(parent_path, tag_name):
         _validate_tag_name(tag_name)
         tag_data = read_file(parent_path, tag_name)
-        return RunTag(tag_name, str(tag_data))
+        if len(tag_data) == 0:
+            raise Exception("Tag '%s' is malformed. No data found." % tag_name)
```

Comment:
Why not allow empty tags too? It seems fine to do that, especially as many of these will be automatically generated by code.